modules/op-running-pipelines-in-restricted-environment.adoc

```diff
@@ -0,0 +1,163 @@
++
+[source,terminal]
+----
+$ oc describe imagestream python -n openshift
+----
++
+.Example output
+[source,terminal]
+----
+Name:			python
+Namespace:		openshift
+[...]
+
+3.8-ubi8 (latest)
+  tagged from registry.redhat.io/ubi8/python-38:latest
+    prefer registry pullthrough when referencing this tag
+
+  Build and run Python 3.8 applications on UBI 8. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-python-container/blob/master/3.8/README.md.
+  Tags: builder, python
+  Supports: python:3.8, python
+  Example Repo: https://github.com/sclorg/django-ex.git
+
+  ! error: Import failed (InternalError): Internal error occurred: registry.redhat.io/ubi8/python-38:latest: Get "https://registry.redhat.io/v2/": Forbidden
+
+[...]
+----
+
+. Mirror the supported image tag to the private registry:
++
+[source,terminal]
+----
+$ oc image mirror registry.redhat.io/ubi8/python-38:latest $MIRROR_REG/ubi8/python-38
+----
+
+. Import the image:
++
+[source,terminal]
+----
+$ oc tag $MIRROR_REG/ubi8/python-38 python:latest --scheduled
+----
++
+You must periodically re-import the image. The `--scheduled` flag enables automatic re-import of the image.
+
+. Verify that the images with the given tag have been imported:
++
+[source,terminal]
+----
+$ oc describe imagestream python -n openshift
+----
++
+.Example output
++
+[source,terminal]
+----
+Name:			python
+Namespace:		openshift
+[...]
+
+3.8-ubi8 (latest)
+ tagged from <mirror-registry>:<port>/ubi8/python-38:latest
+   prefer registry pullthrough when referencing this tag
+
+ Build and run Python 3.8 applications on UBI 8. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-python-containe
+r/blob/master/3.8/README.md.
+ Tags: builder, python
+ Supports: python:3.8, python
+ Example Repo: https://github.com/sclorg/django-ex.git
+
+ * registry.redhat.io/ubi8/python-38@sha256:3ee3c2e70251e75bfeac25c0c33356add9cc4abcbc9c51d858f39e4dc29c5f58
+
+[...]
+----
+
+. For the back-end interface, `vote-api`, verify that the required images tag is not imported:
++
+[source,terminal]
+----
+$ oc describe imagestream golang -n openshift
+----
++
+.Example output
++
+[source,terminal]
+----
+Name:			golang
+Namespace:		openshift
+[...]
+
+1.14.7-ubi8 (latest)
+  tagged from ec2-18-217-137-132.us-east-2.compute.amazonaws.com:5000/ubi8/go-toolset:1.14.7
+    prefer registry pullthrough when referencing this tag
+
+  Build and run Go applications on UBI 8. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/golang-container/blob/master/README.md.
+  Tags: builder, golang, go
+  Supports: golang
+  Example Repo: https://github.com/sclorg/golang-ex.git
+
+  ! error: Import failed (NotFound): dockerimage.image.openshift.io "ec2-18-217-137-132.us-east-2.compute.amazonaws.com:5000/ubi8/go-toolset:1.14.7" not found
```

Comment:
I would omit this line, it's not important for this doc and might scary the reader.